editor/scene_tree_dock.cpp

```diff
@@ -419,6 +419,9 @@ void SceneTreeDock::_tool_selected(int p_tool, bool p_confirm_override) {
 
 			if (!selected_item) {
 				selected_item = tree->get_root();
+				if (!selected_item) {
+					break;
```

Comment:
`return;` might be a better choice here, I'm not sure.

Reply:
I can do that. The reason I used `break` was for consistency with how it's done in other `case` blocks in that function.

Reply:
`break` is used everywhere in this function, so it's ok to leave it.